ubuntu/create_deb.py

```diff
@@ -38,8 +38,12 @@
 	shutil.copy(exec_file,usr_bin)
 	shutil.copy(desk_file,usr_share)
 	shutil.copytree(src_dir,usr_share_kawaii)
-	subprocess.call(['dpkg-deb','--build',dest_dir])
-	deb_pkg = os.path.basename(dest_dir)+'.deb'
-	print('deb package created successfully in current directory. Now install the package using command: \n\nsudo gdebi {0}\n\n'.format(deb_pkg))
+	dpkg_errcode = subprocess.call(['dpkg-deb','--build',dest_dir])
+	deb_pkg = './'+os.path.basename(dest_dir)+'.deb'
+	if not dpkg_errcode:
```

Comment:
I'll switch to `.exists()`, but otherwise I separated the ifs because the error message is if the file doesn't exist for some reason.

I originally made another error message if dpkg-deb failed for the if you see right now, but it seemed redundant because dpkg-deb already outputs an error, so create_deb also spitting out another error was pointless

Reply:
I added a dpkg-deb specific error to this if, I hope that resolves your concerns.

This means there are now two errors:
* dpkg-deb fails
* can't find .deb